front/components/assistant/AssistantDetails.tsx

```diff
@@ -276,7 +276,7 @@ export function AssistantDetails({
               <div className="flex items-center gap-2">
                 <Icon visual={PlanetIcon} size="xs" />
                 <div>
-                  `Assistant can navigate the web (browse any provided links,
+                  Assistant can navigate the web (browse any provided links,
                   make a google search, etc.) answer
```

Comment:
nice catch; we can also edit the sentence below to make sense
```suggestion
                  make a google search, etc.) to answer
```